zipline/pipeline/loaders/quarter_estimates.py

```diff
@@ -0,0 +1,287 @@
+import pandas as pd
+from six import viewvalues
+from toolz import groupby
+from zipline.pipeline.common import (
```

Comment:
there should be one line of whitespace between the third party imports and internal imports.